datacapture/src/main/java/com/google/android/fhir/datacapture/fhirpath/FHIRPathEngineHostServices.kt

```diff
@@ -26,8 +26,7 @@ import org.hl7.fhir.r4.utils.FHIRPathEngine
  */
 internal object FHIRPathEngineHostServices : FHIRPathEngine.IEvaluationContext {
   override fun resolveConstant(appContext: Any?, name: String?, beforeContext: Boolean): Base? {
```

Comment:
optional : can you please also check below suggestion.
```suggestion
 override fun resolveConstant(appContext: Any?, name: String?, beforeContext: Boolean) = (appContext as? Map<*, *>)?.get(name) as? Base
```

Reply:
I'm ok w this